package.json

```diff
@@ -45,18 +45,23 @@
   },
   "version": "9.2.0-next.0",
   "dependencies": {
+    "@angular/benchpress": "^0.2.0",
+    "@angular/dev-infra": "file:../dev-infra",
```

Comment:
This is referring to a locally published version of dev-infra. Once PR [#36800](https://github.com/angular/angular/pull/36800) is submitted and once Issue [#36986](https://github.com/angular/angular/issues/36986) is resolved, this will be changed to point to the actual dev-infra.

Reply:
Currently only dependencies actually needed for the release output of Angular Components are in `dependencies`. I think for consistency we should put all new dependencies into `devDependencies`.

<sup>I'm aware that this doesn't matter for release packages; but we should preserve it for consistency I guess</sup>